### PR TITLE
clean up dev guide

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

File: docs-src/devguide.md [hunk 1]
```diff
@@ -50,43 +55,36 @@ explicitly decoupling ourselves from the Kubernetes API versioning cycle to
...
-* 1H 2020: Monthly release cycle, with first release targeted for January 31
-* 2H 2020: Slower release cycle
-
 ## Building the code
 
-The project uses `make` to drive the build. `make` will build the manager binary, run code generators, and run static
-analysis against the code. You can kick off an overall build from the top-level makefile:
+The project uses `make` to drive the build.
+`make` will run code generators, and run static analysis against the code and
+generate k8s CRDs.
```
Comment (robscott) [inline docs-src/devguide.md:62]: I think I'd prefer to use "Kubernetes" here just for clarity.

File: docs-src/devguide.md [hunk 2]
```diff
@@ -50,43 +55,36 @@ explicitly decoupling ourselves from the Kubernetes API versioning cycle to
...
 make
 ```
 
-## Testing the code
+## Install CRDs
 
-The easiest way to test the code is to use the [kubebuilder][kubebuilder]
-created CRD with a [kind][kind] cluster. Follow the installation instructions
-for `kind` in the README in the repo.
+To install service-apis CRDs into a k8s cluster:
```
Comment (robscott) [inline docs-src/devguide.md:71]: I think I'd prefer to use "Kubernetes" here just for clarity.

File: docs-src/devguide.md [hunk 3]
```diff
@@ -50,43 +55,36 @@ explicitly decoupling ourselves from the Kubernetes API versioning cycle to
...
 
 ```shell
-kind create cluster
-...
-# Install the CRDs
 make install
+```
 
-# Remove the CRDs and associated CRs
+To uninstll CRDs and associated resources:
```
Comment (robscott) [inline docs-src/devguide.md:77]: uninstall

File: docs-src/devguide.md [hunk 4]
```diff
@@ -37,11 +40,13 @@ for this project:
 * [Open issues][gh-issues]
 * [Project dashboard][gh-dashboard]
 
-[gh-issues]: https://github.com/kubernetes-sigs/service-apis/issues
-[gh-dashboard]: https://github.com/kubernetes-sigs/service-apis/projects/1
-
 Issues labeled `good first issue` and `help wanted` are especially good for a
 first contribution.
+Please take a look at [milestones][gh-milestones] as well.
```
Comment (robscott) [inline docs-src/devguide.md:45]: Not quite sure how to say this, but I think you could provide a bit more detail here.

```suggestion
We use [milestones][gh-milestones] to track our progress towards releases. Looking at our current milestone can help identify our highest priority issues.
```

Should we also link to releases here?

**Review by robscott — COMMENTED** [review]
Thanks for your work on this!

**Comment by hbagdi** [comment]
@robscott Updated. PTAL.

**Comment by robscott** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**